### PR TITLE
Remove Flask related documents

### LINKED ISSUE #12: Clarify the lack of FlaskMiddleware for Python
Currently the implementation of `FlaskMiddleware` does not exist in the repository whereas it is mentioned in [the document](https://github.com/open-telemetry/opentelemetry-sqlcommenter/tree/main/docs/python/flask).

To avoid unnecessary confusion, we need either of the following options:

1. Remove entire document contents regarding `FlaskMiddleware` for now from the repository
2. Change the doc to explicitly mention `FlaskMiddleware` is not available in the current version

I would suggest to take option 1 because the document content can be recovered from the history afterwards when it is necessary.

### PR DESCRIPTION
**Pull request by ymotongpoo**
Fixes #12

### CODE REVIEW

**Review by sjs994 — COMMENTED** [review]
Thanks Yoshi for the PR,

The image `docs/images/flask-logo.png` is linked with other places like [psycopg2](https://open-telemetry.github.io/opentelemetry-sqlcommenter/python/psycopg2/#with-flask) and [sqlalchemy](https://open-telemetry.github.io/opentelemetry-sqlcommenter/python/SQLAlchemy/#with-flask) page.

**Comment by ymotongpoo** [comment]
@sjs994 Thanks for pointing it out. I also removed the section that mentions non-existing features for Flask.

**Comment by sjs994** [comment]
Thanks Yoshi,

I just checked now, there seems to be another way to instrument flask with [sqlalchemy](https://github.com/open-telemetry/opentelemetry-sqlcommenter/blob/d2de49dd0a3f6633239bb7ab42a346c8e70025f3/python/sqlcommenter-python/google/cloud/sqlcommenter/sqlalchemy/executor.py#L57) and [psycopg2](https://github.com/open-telemetry/opentelemetry-sqlcommenter/blob/d2de49dd0a3f6633239bb7ab42a346c8e70025f3/python/sqlcommenter-python/google/cloud/sqlcommenter/psycopg2/extension.py#L68).

Should we update that in the document ?

**Comment by ymotongpoo** [comment]
@sjs994 Thanks for checking it. I should have checked the implementation more thoroughly. With your comment, I realized that the issue is that sqlcommenter is just lacking the implementation `FlaskMiddleware` and auto detection of some attributes by `get_flask_info()` is available. I'll take this into this PR.

**Comment by ymotongpoo** [comment]
@sjs994 Sorry for delayed update! I made some changed to match the description with the current implementations.

File: docs/python/SQLAlchemy/README.md [hunk 1]
```diff
@@ -190,10 +189,7 @@ Examining our Postgresql server logs
 ```
 
 ### With flask
-When coupled with the web framework [flask](http://flask.pocoo.org), we still provide middleware to correlate
-your web applications with your SQL statements from psycopg2. Please see this end-to-end guide below:<br>
-[![](../../images/flask-logo.png)](../flask/#with-psycopg2)
-
+When coupled with the web framework [flask](http://flask.pocoo.org), we sprovide a class (`BeforeExecuteFactory`) to correlate your web applications with your SQL statements from psycopg2.
```
Comment (sjs994) [inline docs/python/SQLAlchemy/README.md:192]: typo: `provide`

File: docs/python/SQLAlchemy/README.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
 # SQLAlchemy
 ![](../../images/sqlalchemy-logo.png)
- 
+
 - [Introduction](#introduction)
 - [Requirements](#requirements)
 - [BeforeExecuteFactory](#BeforeExecuteFactory)
 - [Fields](#fields)
 - [End to end examples](#end-to-end-examples)
-- [With flask](#with-flask)
```
Comment (sjs994) [inline docs/python/SQLAlchemy/README.md:9]: I think we have a flask section ?

File: docs/python/psycopg2/README.md [hunk 3]
```diff
@@ -11,7 +11,6 @@
     - [CommenterCursorFactory](#commentercursorfactory)
       - [with_openCensus=True](#with_opencensustrue)
     - [Expected fields](#expected-fields)
-      - [Default options with flask](#default-options-with-flask)
```
Comment (sjs994) [inline docs/python/psycopg2/README.md:14]: This section is also present in the doc.

File: docs/python/psycopg2/README.md [hunk 4]
```diff
@@ -27,7 +26,6 @@
         - [With DB API Thread Safety](#with-db-api-thread-safety-1)
         - [With Driver Parameter Style](#with-driver-parameter-style-1)
         - [With libpq Version](#with-libpq-version-1)
-    - [With flask](#with-flask)
```
Comment (sjs994) [inline docs/python/psycopg2/README.md:30]: this section is also present.

**Review by sjs994 — COMMENTED** [review]
Thanks Yoshi, few minor comments here.

- This PR completely removes the flask/README.md file. That might give users an indication that flask is not supported as a whole unless someone opens up SQLAlchemy/psycopg2 links.
- Should we have the flask logo attached to corresponding sections in SQLAlchemy and psycopg2 pages (which has been removed in this PR) ?

File: docs/python/psycopg2/README.md (hunk 4 above)
Comment (ymotongpoo) [reply docs/python/psycopg2/README.md:30]: Thank you for catching this!

File: docs/python/psycopg2/README.md (hunk 3 above)
Comment (ymotongpoo) [reply docs/python/psycopg2/README.md:14]: Thanks for catching this!

File: docs/python/SQLAlchemy/README.md (hunk 2 above)
Comment (ymotongpoo) [reply docs/python/SQLAlchemy/README.md:9]: Yes, we do! Thanks!

File: docs/python/SQLAlchemy/README.md (hunk 1 above)
Comment (ymotongpoo) [reply docs/python/SQLAlchemy/README.md:192]: Oops! Thanks!

**Comment by ymotongpoo** [comment]
@sjs994 Thanks for the comment. I agree your points. I reverted the Flask logo into the READMEs of psycopg2 and SQLAlchemy .

File: docs/python/psycopg2/README.md [hunk 5]
```diff
@@ -27,7 +27,7 @@
...
+        - [With Flask](#with-flask)
```
Comment (sjs994) [inline docs/python/psycopg2/README.md:30]: This extra space would bring the flask link along the lines of `With libpq version`, which comes under `Results` section (https://open-telemetry.github.io/opentelemetry-sqlcommenter/python/psycopg2/#with-flask).

**Review by sjs994 — COMMENTED** [review]
Easy way to verify the document changes is to create a gh-pages action in your repo and point it to `docs` folder. So, things can be verified and fixed quickly.

File: docs/python/psycopg2/README.md (hunk 5 above)
Comment (sjs994) [reply docs/python/psycopg2/README.md:30]: @ymotongpoo Thanks for the PR!. Except this small indentation, things looks good to me.

Comment (ymotongpoo) [reply docs/python/psycopg2/README.md]: @sjs994 Oops, it was totally mistake. I misunderstood the section level. Fixed!

**Review by sjs994 — APPROVED** [review]
Thanks @ymotongpoo. Looks Good to me.

**Pull request merged**